apps/passport-server/src/services/telegramService.ts

```diff
@@ -0,0 +1,125 @@
+import { EdDSATicketPCD, EdDSATicketPCDPackage } from "@pcd/eddsa-ticket-pcd";
+import { Bot, InlineKeyboard } from "grammy";
+import { fetchTelegramConversation } from "../database/queries/telegram/fetchTelegramConversation";
+import { insertTelegramConversation } from "../database/queries/telegram/insertTelegramConversation";
+import { markTelegramUserAsVerified } from "../database/queries/telegram/updateTelegramConversation";
+import { ApplicationContext } from "../types";
+import { logger } from "../util/logger";
+
+export class TelegramService {
+  private context: ApplicationContext;
+  private bot: Bot;
+  private privateChatId: string;
+
+  public constructor(
+    context: ApplicationContext,
+    bot: Bot,
+    privateChatId: string
+  ) {
+    this.context = context;
+    this.bot = bot;
+    this.privateChatId = privateChatId;
+
+    this.bot.command("start", async (ctx) => {
+      await insertTelegramConversation(this.context.dbPool, {
+        telegram_chat_id: ctx.chat.id,
+        telegram_user_id: ctx.message!.from.id
+      });
+      ctx.reply(
+        "Welcome! 👋\n\nPlease verify your credentials via the Passport app, so we can add you to the correct channels.",
+        {
+          reply_markup: new InlineKeyboard().url(
+            "Verify with Passport 🚀",
+            encodeURI(
+              `https://staging.pcdpass.xyz/#/get-without-proving?request=${JSON.stringify(
```

Comment:
you should use the `PASSPORT_CLIENT_URL` environment here rather than hard coding `staging` or not `staging` here

Reply:
PASSPORT_CLIENT_URL doesn't seem to exist!

Reply:
Ok, it does now